src/Themes.ts

```diff
@@ -1,226 +1,319 @@
-        DOMCacheGetOrSet('platonicUpgradePics').style.backgroundColor = '#0c0c0f'
+};
 
-        themeButton.textContent = 'Lighter Dark Mode';
+interface Theme {
+    green: string
+    red: string
+    crimson: string
+    orchid: string
+    bar: {
+        red: string
+        yellow: string
+        green: string
     }
-    if (current === 'Lighter Dark Mode') { //Switches to 'Light Mode' (Might be reworked soon)
-        document.body.style.setProperty('--bg-color', '#7c7891');
-        document.body.style.setProperty('--alert-color', '#444250');
-        document.body.style.setProperty('--history-lines', '#156e71');
-        document.body.style.setProperty('--box-color', '#444151');
-        document.body.style.setProperty('--boxmain-bordercolor', '#d894d8');
-        document.body.style.setProperty('--button-color', '#136062');
-        document.body.style.setProperty('--hover-color', '#187c7f');
-        document.body.style.setProperty('--buttonbuy-color', '#4c495a');
-        document.body.style.setProperty('--buildings-canbuy-color', '#7c7891');
-        document.body.style.setProperty('--buildings-hover-color', '#9794a8');
-        document.body.style.setProperty('--blessings-canbuy-color', '#747088');
-        document.body.style.setProperty('--blessings-hover-color', '#858298');
-        document.body.style.setProperty('--tab-color', '#105254');
-        document.body.style.setProperty('--singtab-color', '#00d');
-        document.body.style.setProperty('--hoversing-color', '#1052B6');
-        DOMCacheGetOrSet('corruptionStatsLoadouts').style.backgroundColor = '#444151'
-        DOMCacheGetOrSet('corruptionStatsLoadouts').style.borderColor = '#dd8f00'
-        DOMCacheGetOrSet('actualPotionShop').style.backgroundColor = '#444250'
-        DOMCacheGetOrSet('actualPotionShop').style.borderColor = '#dd0'
-        DOMCacheGetOrSet('actualShop').style.backgroundColor = '' //Remove inline CSS
-        DOMCacheGetOrSet('actualShop').style.borderColor = ''
-        DOMCacheGetOrSet('platonicUpgradePics').style.backgroundColor = ''
+}
 
-        DOMCacheGetOrSet('achievementcolorcode2').style.color = '#ff01f3' //To make easier to read text (Maybe clean up)
-        DOMCacheGetOrSet('corruptionDescription').style.color = '#ff01f3'
-        DOMCacheGetOrSet('corruptionTesseractsValue').style.color = '#ff01f3'
-        DOMCacheGetOrSet('corruptionTotalScore').style.color = '#ad5ad7'
-        DOMCacheGetOrSet('corruptionAntExponentValue').style.color = '#ad5ad7'
-        DOMCacheGetOrSet('corruptionIntroduction').style.color = '#ad5ad7'
-        DOMCacheGetOrSet('antwelcome').style.color = 'darkgrey'
+export const toggleTheme = (initial = false, themeNumber = 1, change = false) => {
+    const themeButton = DOMCacheGetOrSet('theme');
+    const body = document.body;
+    const { bar } = theme;
 
-        themeButton.textContent = 'Light Mode';
+    if (change) {
+        themeNumber;
```

Comment:
```suggestion

```

Reply:
Wait I removed it... It reapeared on last squish...